keel-sql/src/test/kotlin/com/netflix/spinnaker/keel/tx/DeliveryConfigDeletionTests.kt

```diff
@@ -24,16 +23,8 @@ import strikt.assertions.isEmpty
 @ExtendWith(SpringExtension::class)
 @SpringBootTest(
   classes = [KeelApplication::class, TestConfiguration::class],
-  webEnvironment = MOCK,
-  properties = [
-    "spinnaker.baseUrl=http://spinnaker",
-    "sql.enabled=true",
-    "sql.connection-pools.default.jdbc-url=jdbc:tc:mysql:5.7.22://somehostname:someport/databasename",
-    "sql.migration.jdbc-url=jdbc:tc:mysql:5.7.22://somehostname:someport/databasename",
-    "spring.datasource.driver-class-name=org.testcontainers.jdbc.ContainerDatabaseDriver"
-  ]
```

Comment:
I just made all these properties consistent in `keel-sql/src/test/resources/application.properties` there's no need to specify them in each class as they're always the same.